features/webgl2-context-colorspace.yml

```diff
@@ -0,0 +1,6 @@
+name: WebGL2 Color Space
```

Comment:
I'm open to splitting this into two features, one has baseline low, the other is only supported in Chrome/Edge.

It also has the same support versions as the same functionality in WebGL1, so we could alternatively merge it that way.